src/v/cluster/types.cc

```diff
@@ -444,6 +444,104 @@ struct adl<security::acl_binding> {
     }
 };
 
+/*
```

Comment:
I think we have a unit test called `roundtrip_adl` or smth like that in the tree that would be useful here.

Reply:
Ill add this